docs/db-schema.md

```
@@ -0,0 +1,228 @@
+# DB Schema 명세
+
+## ERD 개요
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)